### DISCUSSION TITLE
Is matches-css supposed to fail when the value is "transparent" or a variable?

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#2209] [Q&A] [answered]
I was just looking into making a new ebay sponsored filter as per a reddit request, but it looks like the only difference between sponsored and non-sponsored results is that the non-sponsored ones have "Sponsored" text set to "transparent", while the actual sponsored items use a variable.

I tried using `:matches-css()` with these values, but picker said 0 results.

Any ideas how to address this?

I know they use different classes, but I've no way of knowing if they're the same classes for everybody or if they're changing periodically.

**Comment by ghost** [comment, marked as answer]
Are you sure getComputedStyle returns transparent? matches-css is tricky, because it works only on values returned by getComputedStyle, not what inspector is showing.
